.env.dist

```diff
@@ -7,3 +7,8 @@ GROUP_ID=0
 WEB_PORT=443
 SSL_CRT=./nginx/certs/ssl.crt.dist
 SSL_KEY=./nginx/certs/ssl.key.dist
+FILE_WEB_PORT=
+FILE_DOMAIN=
+FILE_SSL_CRT=
```

Comment:
Can you please include the default certs here just as it was done with the API server?